build.gradle

```diff
@@ -267,6 +267,31 @@ task clean {
     dependsOn cleanLocalMavenRepos
 }
 
+task manualClean {
+    description = 'Clean build files manually, which means that this task works without clean tasks'
+    group = 'Clean'
+
+    doLast {
+        // clean 'build' directories
+        exec {
+            workingDir "${rootDir}"
+            commandLine 'find', '.', '-type', 'd', '-name', 'build', '-print', '-exec', 'rm', '-rf', '{}', ';', '-prune'
```

Comment:
`-delete` is only deleting the contents of the directory, while your `-exec rm` deletes the directory (and its contents).